docs/changelog.rst

```diff
@@ -2,6 +2,8 @@
 Changelog
 =========
 
+* :bug:`3714` Airdrops section will now handle errors retrieving information.
```

Comment:
Change this. The user-facing bug affects windows users of rotki v1.22.0

`Airdrops section will now work again for Windows users`